pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: crypto
-version: 2.1.1+1
+version: 2.1.1+2
```

Comment:
Bug fixes should be a patch version, not a build version.

Reply:
Also include a changelog entry?

Also, behavior change should cause `2.1.2`

Reply:
This definitely requires a bump in version. The old behavior causes HMAC-SHA512 to produce wrong values. So it's best to let people know that they should use 2.1.2 from now on.